pom.xml

```diff
@@ -397,6 +398,20 @@
                 <version>${log4j.version}</version>
             </dependency>
             <dependency>
+              <!-- This is not slf4j's version because of performance concerns
+              http://www.slf4j.org/api/org/slf4j/bridge/SLF4JBridgeHandler.html
+              Please make sure to do performance tests before switching this to slf4j
```

Comment:
maybe also add a note that if someone wants to switch to slf4j they should use logback, which alleviates the problem that jul-to-slf4j has thanks to the [LevelChangePropagator](http://logback.qos.ch/manual/configuration.html#LevelChangePropagator)